bitnami/metallb/Chart.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: v2
+name: metallb
+description: The Metal LB for Kubernetes
+appVersion: 0.8.3
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+keywords:
+  - "load-balancer"
+  - "balancer"
+  - "lb"
+  - "bgp"
+  - "arp"
+  - "vrrp"
+  - "vip"
+home: https://metallb.universe.tf
+icon: https://metallb.universe.tf/images/logo.png
+sources:
+- https://github.com/metallb/metallb
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.3
+maintainers:
+  - name: cellebyte
+    url: https://github.com/Cellebyte
```

```suggestion
apiVersion: v1
name: metallb
description: The Metal LB for Kubernetes
appVersion: 0.8.3
keywords:
  - "load-balancer"
  - "balancer"
  - "lb"
  - "bgp"
  - "arp"
  - "vrrp"
  - "vip"
home: https://metallb.universe.tf
icon: https://metallb.universe.tf/images/logo.png
sources:
  - https://github.com/metallb/metallb
  - https://github.com/bitnami/bitnami-docker-metallb
version: 0.1.3
maintainers:
  - name: cellebyte
    url: https://github.com/Cellebyte
  - name: Bitnami
    email: containers@bitnami.com
```

What do you think about the above changes? Mainly add maintainer, source, remove comments and apiVersion: v1 to be consistent with the other charts in the catalog